lib/features/mailbox_dashboard/presentation/mailbox_dashboard_controller.dart

```diff
@@ -20,6 +21,8 @@ class MailboxDashBoardController extends BaseController {
   final userProfile = Rxn<UserProfile>();
 
   Session? sessionCurrent;
+  MailboxId? sentMailboxId;
+  MailboxId? outboxMailboxId;
```

Comment:
IMO, you should store a map to default mailboxes. In the future, may be other mailboxes need to be used. Fx: `Draft` -> when composing email.

Reply:
If default save, where does mailboxId get? Because it is taken from the server when there is a network. What if they change every time they take them? Please give more clear solution.

Reply:
I mean we use Map<role, mailboxId> to store default mailbox here instead of separate it in each properties